tests/dev/test_clean_generated_output.py

```
@@ -100,3 +103,39 @@ def test_clean_paths_removes_nested_empty_generated_directories(tmp_path: Path)
+    evidence_dir = repo / "output" / "evidence"
+    scratch_dir = repo / "output" / "scratch"
+    evidence_dir.mkdir(parents=True)
+    scratch_dir.mkdir(parents=True)
+    tracked_evidence = evidence_dir / "summary.json"
+    ignored_generated = evidence_dir / "run.log"
+    untracked_generated = scratch_dir / "scratch.txt"
+    tracked_evidence.write_text('{"claim": "preserved"}\n', encoding="utf-8")
+    ignored_generated.write_text("local log\n", encoding="utf-8")
+    untracked_generated.write_text("temporary\n", encoding="utf-8")
+    (repo / ".gitignore").write_text("output/evidence/*.log\n", encoding="utf-8")
+    subprocess.run(["git", "init", "-q"], cwd=repo, check=True)
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)